torchvision/prototype/transforms/_geometry.py

```diff
@@ -155,12 +155,12 @@ class FiveCrop(Transform):
     """
     Example:
         >>> class BatchMultiCrop(transforms.Transform):
-        ...     def forward(self, sample: Tuple[Tuple[features.Image, ...], features.Label]):
-        ...         images, labels = sample
-        ...         batch_size = len(images)
-        ...         images = features.Image.wrap_like(images[0], torch.stack(images))
+        ...     def forward(self, sample: Tuple[Tuple[Union[features.Image, features.Video], ...], features.Label]):
+        ...         images_or_videos, labels = sample
+        ...         batch_size = len(images_or_videos)
+        ...         images_or_videos = features.Image.wrap_like(images_or_videos[0], torch.stack(images_or_videos))
```

Comment:
```suggestion
        ...         image_or_video = images_or_videos[0]
        ...         images_or_videos = type(image_or_video).wrap_like(image_or_video, torch.stack(images_or_videos))
```

Reply:
Sorry I forced pushed and lost this change. Let me reapply.